frontend/src/pages/team/Projects.vue

```diff
@@ -18,11 +18,12 @@
                 </ff-button>
             </div>
         </template>
-        <template v-else>
+        <template v-else-if="!loading">
```

Comment:
This condition needs moving to the top so it takes precedence over the others. As it stands, it is never being reached.